test/cypress/integration/settings/your-psp-stripe-tasklist.cy.test.js

```diff
@@ -175,4 +176,30 @@ describe('Your PSP Stripe page', () => {
       cy.get('strong[id="task-vatNumber-status"]').should('contain', 'complete')
     })
   })
+
+  describe('Company number task', () => {
+    it('should click company registration task and display page correctly', () => {
+      setupYourPspStubs()
+      cy.get('span').contains('Company registration number').click()
+      cy.get('h1').should('contain', 'Company registration number')
+    })
+
+    it('should redirect back to the task List when a valid company registration number is submitted', () => {
```

Comment:
As mentioned in the tech team meeting, as this `it` is a continuation of the previous `it`, it would be better to combine them into one test as the second test currently relies on the state left behind by the first test as per https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests